test/k8sT/Policies.go

```diff
@@ -2340,6 +2340,120 @@ var _ = SkipDescribeIf(func() bool {
 				"%q Clusterwide Policy cannot be deleted", ingressDenyAllPolicy)
 		})
 	})
+
+	//TODO: Check service with IPV6
```

Comment:
Should we file an issue here given that we've had previous discussions to convert TODOs in code to issues?

Reply:
Good point! I'm going to file the issue to move the whole suite to a unit test :-)